README.rst

```diff
@@ -0,0 +1,34 @@
+
+
+==================
+Bioelectrics: Monodomain problem on a 2D domain with simple HodgkinHuxley1952 cell model
+==================
+
+Running the example
+===================
+Activate the openCMISS virtual environment for python bindings
+  source <path-to-opencmiss>/install/virtual_environments/oclibs_venv_py27_release/bin/activate
+
+Run the python script
+  cd ./src/Python/
+
+  python monodomain_2D_HH.py
+
+To run in parallel
+  cd ./src/Python/
+
+  mpirun -np <number-of-threads> python monodomain_2D_HH.py
```

Comment:
number-of-processors

mpiexec -n <number-of-processors> python monodomain_2D_HH.py